pkg/cmd/issue/view/view.go

```diff
@@ -161,22 +196,102 @@ func printHumanIssuePreview(io *iostreams.IOStreams, issue *api.Issue) error {
 	}
 
 	// Body
-	if issue.Body != "" {
-		fmt.Fprintln(out)
-		style := markdown.GetStyle(io.TerminalTheme())
-		md, err := markdown.Render(issue.Body, style, "")
+	fmt.Fprintln(out)
+	if issue.Body == "" {
+		issue.Body = "_No description provided_"
+	}
+	style := markdown.GetStyle(io.TerminalTheme())
+	md, err := markdown.Render(issue.Body, style, "")
+	if err != nil {
+		return err
+	}
+	fmt.Fprint(out, md)
+	fmt.Fprintln(out)
+
+	// Comments
+	if issue.Comments.TotalCount > 0 {
+		comments, err := issueComments(io, issue.Comments)
 		if err != nil {
 			return err
 		}
-		fmt.Fprintln(out, md)
+		fmt.Fprint(out, comments)
 	}
-	fmt.Fprintln(out)
 
 	// Footer
-	fmt.Fprintf(out, cs.Gray("View this issue on GitHub: %s\n"), issue.URL)
+	fmt.Fprintf(out, cs.Gray("View this issue on GitHub: %s"), issue.URL)
+
 	return nil
 }
 
+func issueComments(io *iostreams.IOStreams, comments api.IssueComments) (string, error) {
+	var b strings.Builder
+	cs := io.ColorScheme()
+	retrievedCount := len(comments.Nodes)
+	hiddenCount := comments.TotalCount - retrievedCount
+
+	if hiddenCount > 0 {
+		fmt.Fprint(&b, cs.Gray(fmt.Sprintf("———————— Hiding %v comments ————————", hiddenCount)))
+		fmt.Fprintf(&b, "\n\n\n")
+	}
+
+	for i, comment := range comments.Nodes {
+		last := i+1 == retrievedCount
+		cmt, err := issueComment(io, comment, last)
+		if err != nil {
+			return "", err
+		}
+		fmt.Fprint(&b, cmt)
+		if last {
+			fmt.Fprintln(&b)
+		}
+	}
+
+	if hiddenCount > 0 {
+		fmt.Fprint(&b, cs.Gray("Use --comments to view the full conversation"))
+		fmt.Fprintln(&b)
+	}
+
+	return b.String(), nil
+}
+
+func issueComment(io *iostreams.IOStreams, comment api.IssueComment, newest bool) (string, error) {
```

Comment:
This method is called just `issueComment`. Could it be named more descriptively to describe what does it do?